docs/strategy-callbacks.md

```diff
@@ -510,6 +510,8 @@ Each of these methods are called right before placing an order on the exchange.
 !!! Note
     If your custom pricing function return None or an invalid value, price will fall back to `proposed_rate`, which is based on the regular pricing configuration.
 
+!!! Note
+    Within this function you will have access to the Trade object, giving you the flexibility to adjust your current entry or exit price in relation to the current trade status. This object is available as soon as the first entry order associated with the trade is created.
```

Comment:
I think wording of this note is quite ... non-specific
This documentation segment covers both entry and exit functions - however this limitation only applies to the entry function (which is not immediately clear from this).

i'd also word this differently / invert  how it's worded - with the assumption that 70-80% of the users will only use one entry - where this is irrelevant.